server/controllers/articleController.js

```diff
@@ -0,0 +1,5 @@
+// controller for articles endpoint
+
+module.exports = {
+  // addArticle: (req, res) => {},
+};
```

Comment:
Logic should be here 
<img width="1126" alt="Screenshot 2020-04-25 at 6 59 45 PM" src="https://user-images.githubusercontent.com/21286134/80281214-463b7f00-8727-11ea-864e-1b98dc00a1a2.png">